docs/ops/opset.md

```diff
@@ -6,9 +6,9 @@ This topic provides a complete list of available sets of operations supported in
 
 | OpenVINO™ Version | Actual Operations Set            | 
 | :---------------- | :------------------------------- | 
-| 2020.4            | [opset3](./docs/ops/opset3.md)   | 
-| 2020.2            | [opset2](./docs/ops/opset2.md)   | 
-| 2020.1            | [opset1](./docs/ops/opset1.md)   | 
+| 2020.4            | [opset3](opset3.md)   | 
+| 2020.2            | [opset2](opset2.md)   | 
```

Comment:
Let's add a line that OpenVINO version 2020.3 uses opset2 ops.

Reply:
Added